spacy/errors.py

```diff
@@ -913,6 +913,11 @@ class Errors(metaclass=ErrorsWithCodes):
     E1034 = ("Node index {i} out of bounds ({length})")
     E1035 = ("Token index {i} out of bounds ({length})")
     E1036 = ("Cannot index into NoneNode")
+    E1037 = ("Document JSON representation doesn't adhere to schema: {message}")
```

Comment:
To make this more similar to existing error messages:

```suggestion
    E1037 = ("Invalid JSON input: {message}")
```